values/gitea/gitea.gotmpl

```diff
@@ -89,7 +89,8 @@ gitea:
     session:
       COOKIE_SECURE: true
       DOMAIN: {{ $giteaDomain }}
-      
+    webhook:
+      ALLOWED_HOST_LIST: "drone.{{ $v.cluster.domainSuffix }}"
```

Comment:
`ALLOWED_HOST_LIST: '*'` will ensure that gitea can send web hooks to services inside clusters and to services outside the cluster

Reply:
Done, Thanks for your help.